Dockerfile.deploy

```diff
@@ -1,11 +1,9 @@
-FROM fabric8/fabric8-openshift-nginx:vd83b3a1
-MAINTAINER "Pete Muir <pmuir@bleepbleep.org.uk>"
+FROM nginx:latest
 
 USER root
 
-RUN rm -rf /usr/share/nginx/html/
+RUN rm -rf /usr/share/nginx/html
 COPY dist /usr/share/nginx/html
-RUN chmod -R 777 /usr/share/nginx/html/
+RUN chown -R fabric8:fabric8 /usr/share/nginx/html
 
-
-USER ${FABRIC8_USER_NAME}
+USER fabric8
```

Comment:
Why not use `${FABRIC8_USER_NAME}` as is done [here](https://github.com/fabric8-ui/fabric8-ui/search?utf8=%E2%9C%93&q=FABRIC8_USER_NAME&type=)?

Reply:
That env_var is set by the `Dockerfile.builder`. In cases where you want to run the `Dockerfile.deploy` in host without the `Dockerfile.builder` setting up that var (manually update some content in `/dist` post-build for testing, build locally without builder image or something else) then it'll work just fine, as it's primary goal is just to create an image by putting `/dist` contents in `nginx` doc-root.

So the upside is it doesn't need to be dependent on the another Dockerfile to operate.
Downside is, none actually.

> We should IMHO keep a dedicated Dockerfile for deployments and one for building. This way we can control what goes into what build.

That's how it is.